Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-contests.tsx

```diff
@@ -15,19 +22,30 @@ import { useUrlParams } from './common/use-url-params';
 import { PageParams } from '../common/pages-types';
 import { generateCategoryFilters, generateStatusFilters, generateStrategyFilters } from './contests/contest-filter-utils';
 import { areStringEqual } from '../utils/compare-utils';
+import generateSortingStrategy from '../common/contest-sorting-utils';
+import { useCategoriesBreadcrumbs } from './use-contest-categories-breadcrumb';
+import {
+    DEFAULT_FILTER_TYPE,
+    DEFAULT_SORT_FILTER_TYPE,
+    DEFAULT_SORT_TYPE,
+    DEFAULT_STATUS_FILTER_TYPE,
+} from '../common/constants';
 
 interface IContestsContext {
     state: {
         contests: IIndexContestsType[];
         possibleFilters: IFilter[];
+        possibleSorting: ISort[];
```

Comment:
IMO Plural name for the collection sounds better to me. Something like 'possibleSortingFilters' or 'possbileSortingTypes'
@Minkov @shsimeonova any thoughts on this?